sdk/pnpm-workspace.yaml

```diff
@@ -0,0 +1,2 @@
+packages:
+  - 'src'
```

Comment:
I'm good with using a workspace but like we said live, might be best just to start with single package until we have need for multiple

Reply:
sounds good, i'll revert to a more simple setup